viewer/event_db.cpp

```diff
@@ -30,6 +30,17 @@ void EventDB::insert(parser::Event event)
             } 
         }
         _events[klass_id].insert(it.base(), std::move(event));
+        _cache.insert_event(_events[klass_id][_events[klass_id].size() - 1], klass_id);
```

Comment:
I think you can simplify this line: `_events[klass_id][_events[klass_id].size() - 1]` and write this instead: `*_events[klass_id].back()`

Reply:
or you can use return value from the previous line